test/test_utils/get_test_data.jl

```diff
@@ -215,13 +215,18 @@ function build_c_sys5_re(; kwargs...)
             end
             add_time_series!(c_sys5_re, serv, Deterministic("requirement", forecast_data))
         end
-        #for t in 1:2, serv in get_components(ReserveDemandCurve, c_sys5_re)
-        #    add_time_series!(
-        #        c_sys5_re,
-        #        serv,
-        #        PiecewiseFunction("variable", 5, ORDC_cost_ts[t]),
-        #    )
-        #end
+        for (ix, serv) in enumerate(get_components(ReserveDemandCurve, c_sys5_re))
+            forecast_data = SortedDict{Dates.DateTime, TimeArray}()
```

Comment:
Whenever possible I think that we should not use the TimeArray constructor for Deterministic.  It's less efficient and we don't want people to follow the example when the alternative is just as easy.  You could add `TimeSeries.values()` to line 222.